src/pages/CirclesPage/CirclesPage.tsx

```diff
@@ -17,8 +18,6 @@ import { Box, Button, Link, Panel, Text } from 'ui';
 import { Torso } from 'ui/icons';
 import { SingleColumnLayout } from 'ui/layouts';
 
-import { getOrgData } from './getOrgData';
```

Comment:
i'm still looking through this PR, but i wanted to give this comment right away: i think it would be better to write a smaller query for OverviewMenu's needs rather than use this one. There's stuff in this query like epochs and nominees that doesn't need to be fetched for that.